salt/client/ssh/ssh_py_shim.py

```diff
@@ -106,8 +106,10 @@ def need_deployment():
     if os.path.exists(OPTIONS.saltdir):
         shutil.rmtree(OPTIONS.saltdir)
     old_umask = os.umask(0o077)
```

Comment:
Seems this `os.umask` has to be replaced too?
Maybe the correct one would be just:
```python
with salt.utils.files.set_umask(0o077):
    os.makedirs(OPTIONS.saltdir)
```

Reply:
Ah, I see, it's a thin module that shouldn't use utils. Am I right?
Don't we have to mark these lines with `# pylint: disable=...` or `# future pylint: disable=...`?

Reply:
Yes, I need to disable this check here, thanks. And yeah, we don't want to use the utils in the shim because they won't necessarily be available.